common-java-root/gradlew

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env sh
```

Comment:
Does this file need to be checked in?

Reply:
Probably not...I checked in because all the existing root projects had them checked in